adventure.js

```diff
@@ -0,0 +1,53 @@
+var welcome = confirm('Welcome to Shaun of the Dead - Javascript edition!  Ready to Play?');
+if (welcome === true) {
```

Comment:
Not sure if this will ever run. You need to give the user string options (eg "Yes or No"), and then test if `welcome === 'Yes'`